package.json

```diff
@@ -14,7 +14,7 @@
   },
   "scripts": {
     "prerelease": "rimraf artifacts/*",
-    "start": "cross-env NODE_ENV=development webpack --hide-modules --progress --watch --env.browser",
+    "start": "cross-env NODE_ENV=dev webpack --hide-modules --progress --watch --env.browser",
```

Comment:
Changing this would mess up some tooling that will load a specific JS bundle if `NODE_ENV=development`